python/pyproject.toml

```diff
@@ -63,6 +63,7 @@ redis = { version = "^4.6.0", optional = true}
 azure-search-documents = {version = "11.6.0b1", allow-prereleases = true, optional = true}
 azure-core = { version = "^1.28.0", optional = true}
 azure-identity = { version = "^1.13.0", optional = true}
+azure-cosmos = "^4.7.0"
```

Comment:
Shouldn't this be `azure-cosmos = { version = "^4.7.0", optional = true}`?

Reply:
Fixed